src/sentry/tasks/app_store_connect.py

```diff
@@ -0,0 +1,96 @@
+"""Tasks for managing Debug Information Files from Apple App Store Connect.
+
+Users can instruct Sentry to download dSYM from App Store Connect and put them into Sentry's
+debug files.  These tasks enable this functionality.
+"""
+
+import logging
+import tempfile
+from typing import Optional
+
+from sentry.lang.native import appconnect
+from sentry.models import AppConnectBuild, Project, debugfile
+from sentry.tasks.base import instrumented_task
+from sentry.utils.sdk import configure_scope
+
+logger = logging.getLogger(__name__)
+
+
+# Sadly this decorator makes this entire function untyped for now as it does not itself have
+# typing annotations.  So we do all the work outside of the decorated task function to work
+# around this.
+@instrumented_task(name="sentry.tasks.app_store_connect.dsym_download")  # type: ignore
```

Comment:
To make this explicit and consistent with other tasks, let's add the `queue` attribute here.

Reply:
ok, added this back.  Though it is redundant with the `routing_key` arg in `server.py` config.